src/nnet3/nnet-simple-component.cc

```diff
@@ -588,34 +608,64 @@ std::string ClipGradientComponent::Info() const {
          << ", clipping-threshold=" << clipping_threshold_
          << ", clipped-proportion="
          << (count_ > 0 ? static_cast<BaseFloat>(num_clipped_)/count_ : 0);
+  if (self_repair_clipped_proportion_threshold_ != BaseFloat(kUnsetValue))
```

Comment:
There is no need to invoke kUnsetValue anywhere in this code.  Just have those things default to whatever default value you currently use in the code.  The reason for kUnsetValue in NonlinearComponent was that the same base-class was responsible for setting the parameters, but  there were different defaults in different child classes.